src/olympia/activity/tests/test_utils.py

```diff
@@ -303,7 +304,15 @@ def _recipients(self, email_mock):
             assert reply_to.endswith(settings.INBOUND_EMAIL_DOMAIN)
         return recipients
 
-    def _check_email(self, call, url, reason_text):
+    def _check_email(
+        self,
+        call,
+        url,
+        reason_text,
+        author,
```

Comment:
So many non-named arguments make the calls annoying to read. Could you use keyword-only arguments instead (at least for the last 3 you've added), and modify the calls accordingly ?